### DISCUSSION TITLE
Getting rid of ugly icons

### DISCUSSION THREAD

**Discussion by pzant** [qutebrowser/qutebrowser#8228] [General] [answered]
Is it possible to somehow get rid of icons in context menu and other possibles places? Maybe there is at least an opportunity to choose other icons? Until recently, I had not seen them, but after updating to version 3.1.0 they appeared and began to frustrate me.

qutebrowser v3.2.0
Git commit: 
Backend: QtWebEngine 6.7.1
  based on Chromium 118.0.5993.220
  with security patches up to 124.0.6367.202 (plus any distribution patches)
  (source: api)
Qt: 6.7.1

CPython: 3.11.9
PyQt: 6.7.0.dev2404081550

Qt wrapper info:
  PyQt6: success
  PyQt5: not imported
  -> selected: PyQt6 (via autoselect)

**Comment by mschilli87** [comment, marked as answer]
I think the contex menu is completely managed by Qt, so you'd have to raise that issue there and if/once there are paramters to control it (and they are available via PyQt) you can pass them down via qutebrowser's command line arguments. Otherwise, I don't think there is anything the qutebrowser devs could do short of [implementing a custom context menu](https://github.com/qutebrowser/qutebrowser/issues/349).

**Reply by The-Compiler** [reply]
Yep, not much to add there. I don't think there is a way we can tell Qt to not have icons. Maybe a theme (as with qt6ct or somesuch) can do that.
